web/src/api.tsx

```diff
@@ -0,0 +1,104 @@
+import { redirect } from "react-router"
+import { isType } from "./utils"
+
+const api = "/web/"
```

Comment:
Why do you still use "/web" if you can append every endoint to "/"? Your application represents the main configuration. Therefore no "/web" is required.

Reply:
The "web" is required to make configuration easier. If you were to use "/" as an api endpoint you would have to add every URL to the vite proxy which seems like a waste of time.

Reply:
![image](https://github.com/7erra/MANVSim/assets/99326069/9bcca029-e1e0-413d-b157-c3b83b7a58a8)
I cant agree with that. As I was trying to tell you earlier this week, I have found a configuration, where you get rid of the "/web" and dont have to add every sub url as proxy. I had it tested as shown in the picture. On the left you can see the reached debug prints on the console, in the middle you can see the used request urls via react port. On the right you see the current the react proxy configuration used for the request. What is not presented by the Screenshot is the changed configuration of ur api registration server-wise.

![image](https://github.com/7erra/MANVSim/assets/99326069/5147cf79-ae22-48d0-89ea-ddb5911207fd)
This screenshot shows you the negativ example. If you define a specific url after the "/" only this url will be proxied. However using only "/" every url is proxied to the configured url.

I will am open to discussion in person. Additionally I will push the configuration I have made. Link will be posted shortly.